src/ewatercycle/forcing/_hype.py

```diff
@@ -64,19 +75,22 @@ def generate(  # type: ignore
 
         # generate forcing data and retreive useful information
         recipe_output = recipe.run(session=_session(directory))
-        # TODO return files created by ESMValTOOL which are needed by Hype Model
-        # forcing_path = list(recipe_output['...........']).data_files[0]
-        forcing_path = "/foobar.txt"
 
-        forcing_file = Path(forcing_path).name
```

Comment:
the `from pathlib import Path` at the top of the module can be removed. Shall we also add **Hype generate forcing** (e.g. the code block below) to the notebook [generate_forcing.ipynb](https://github.com/eWaterCycle/ewatercycle/blob/main/docs/examples/generate_forcing.ipynb)? 

```python
# Hype
hype_forcing = ewatercycle.forcing.generate(
    target_model="hype",
    dataset="ERA5",
    start_time="1990-01-01T00:00:00Z",
    end_time="1990-12-31T00:00:00Z",
    shape="./data/Rhine/Rhine.shp",
)
print(hype_forcing)
```